server/scripts/standalone/setup-standalone-server.sh

```diff
@@ -58,11 +58,14 @@ function download_package() {
 		local folder=`(cd ${TMP_DOWNLOAD_DIR} && ls)`
 		echo "[setup] Found folder" ${folder}
 
+		if [[ -n ${folder} ]]; then
+			mv ${TMP_DOWNLOAD_DIR}/${folder} ${PLATFORMSDK_DIR}/${out_package_name}
+			echo "[setup] Installed" ${PLATFORMSDK_DIR}/${package_name}
+		else
+			echo "[setup] Failed to install" ${package_name}
```

Comment:
Would it make sense to `exit 1` here?

Reply:
Possibly. It would've helped me, but for experienced users who only want (for example) Xcode 14.2 OR Xcode 15.0, exiting might be frustrating as they would need to download & package unwanted tools/sdks.

Reply:
Good point. Let's leave it as-is